requirements.txt

```diff
@@ -3,3 +3,4 @@ pandas==1.5.*
 boto3==1.19.*
 sodapy==2.1.*
 smartsheet-python-sdk==3.0.*
+geopandas
```

Comment:
does this need to be pinned to anything?

Reply:
glad chia flagged this and yes that seems like a create idea Charlie. maybe stick a comment in here so we don't forget?

Reply:
added a comment thanks y'all!